jsk_recognition_utils/python/jsk_recognition_utils/__init__.py

```diff
@@ -1,12 +1,15 @@
-from jsk_recognition_utils import chainermodels
 from jsk_recognition_utils import color
 from jsk_recognition_utils import conversations
-from jsk_recognition_utils import datasets
 from jsk_recognition_utils import feature
 from jsk_recognition_utils import mask
 from jsk_recognition_utils import visualize
 from jsk_recognition_utils import geometry
 
+try:
+    from jsk_recognition_utils import chainermodels  # NOQA
+    from jsk_recognition_utils import datasets  # NOQA
+except ImportError:
+    print('Please install chainer<7.0.0 to import chainermodels')
```

Comment:
`to import chainermodels` -> `to import chainermodels and datasets`?
(Because we have to notice this message when we face error about `datasets` module as well as `chainermodels` module.)

Reply:
that's true. I will fix it.